examples/hacker_news_assets/hacker_news_assets/assets/__init__.py

```diff
@@ -0,0 +1,45 @@
+import json
+import os
+
+from dagster import load_assets_from_package_module
+from dagster.utils import file_relative_path
+from dagster_dbt.asset_defs import load_assets_from_dbt_manifest
+
+from . import activity_analytics, core, recommender
+
+# TBD: is it a good practice to build asset groups in assets/__init__.py?
+# * pros:
+#       * users can quickly find code even if they are not familiar with the given project
+#       * users can quickly add asset group without thinking about schedules and sensors
+#       * users can easily build asset groups that are cross business domains
+# * cons:
+#       * users will likely need to think about schedules and sensors separately from building assets
```

Comment:
RFC

Reply:
I think it feels a little hidden to put asset code within an __init__ file (I personally wouldn't look there for code) but I support placing assets in their own `assets.py` file for the reasons you pointed out